petclinic-frontend/src/features/inventories/AddInventoryForm.tsx

```diff
@@ -178,7 +178,6 @@ const AddInventoryForm: React.FC<AddInventoryProps> = ({
               id="imageUpload"
               accept="image/*"
               onChange={handleImageUpload}
-              required
```

Comment:
Taking out the 'required' keyword fixes the image bug noticed after the upload image feature implementation. Well spotted.